HarjotSuiingh.md

```diff
@@ -0,0 +1,65 @@
+
+---
+
+name: @HarjotSuiingh
+
+project: E-ink Smart Display
+
+---
+
+
+# Project Name
+
+E-ink Smart Display
+
+
+## Summary
+
+
+What are you going to build? What does it do? Why are you excited to build it?
+
+
+I am going to build an e-ink smart display which is powered by a Rasberry pi. It will be able to act as a display where I can view things such as my calendar and control smart home items that I have whilst also acting as an e-reader making it an all in one device. I am excited for this build because it will allow me to have one device for which I can do everything I need to do which would just make my life simpler.
+
+
+## Plan
+
+
+What steps are you going to take to build it? What tools are you going to use? What will you do first, second, third, etc.?
+
+I will be using the Pi at the start to get everything set up since it is faster and easier to access over ssh. Then I will be moving the project over to the Pico so its permanently connected to thr Screen.
+
+The sensors+accessories will be used to add functionality to the app and allowing me to make it more personalised based on my location and environment
+
+The sd cards will be used to flash the different os' onto the pi
+
+For this project I will be following different tutorials and websites with steps that I have linked below. Essentially, this project will require a Rasberry pi which I already own as well as a Rasberry pico since my pi is a slightly older version and might not be able to run the weather display. I will be connecting the Rasberry pi to the display using a kindle pw due to the display being very high quality and having features such as temperature control. This would be achieved through the kindlepi/kindleberry tutorial. However, I also have an e-ink display which is slightly worse quality I can connect to the pi if I am unable to use the first method as a backup. I will then place this into an ikea frame I already own using some double sided tape and screws to secure the display in the frame. After this I will program the Rasberry pi to be able to display things such as weather. Specifically, I would need to make a rested api on my local network which I will link to my Amazon Alexa through IFTTT.
```

Comment:
what do you mean by temperature control?

Reply:
@bellesea the colour of the screen can be adjusted to make it more yellow / white depending on the environment and lighting conditions so it has the perfect viewing angle and readability. Idk if temperature control is exactly the right word for it but yeah.